internal/memdb/memdb_test.go

```diff
@@ -187,12 +188,14 @@ func TestPluginInit(t *testing.T) {
 		"test",
 		"development",
 		func(ctx context.Context, logger zerolog.Logger, s specs.Destination) (destination.Client, error) {
+			// TODO this won't work because we don't set these back on the spec
```

Comment:
This still needs addressing (we need to be able to change the default batching limits from the dest spec), will do in a later commit.